examples/js/loaders/VRMLLoader.js

```diff
@@ -1050,6 +1050,10 @@ THREE.VRMLLoader.prototype = {
 
 			var line = lines[ i ];
 
+			// The # symbol indicates that all subsequent text, until the end of the line is a comment,
+			// and should be ignored. (see http://www.agocg.ac.uk/train/vrml2rep/part1/guide3.htm)
```

Comment:
Is that link an authority?

I think the first line of your comment is adequate, unless comments are not in the official spec.

Reply:
I've used another link that directly refers to ISO/IEC 14772. This one is definitely trustworthy.